docs/Project.toml

```diff
@@ -1,5 +1,10 @@
 [deps]
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
```

Comment:
VMs don't always have a GPU. Why not run the docs on CPU?

Reply:
I added it to run the documentation (mainly the tutorial) on Atlas.

Reply:
(update) If we remove CUDA, we can't type the `Dense` structure like this:
```julia
struct Dense{
    T <: Union{Matrix{Float32}, CuArray{Float32, 2, CUDA.Mem.DeviceBuffer}}
    Y <: Union{Vector{Float32}, CuArray{Float32, 1, CUDA.Mem.DeviceBuffer}}
    }
  w :: Param{T}
  b :: Param{Y}
```
which makes documentation run fail.

Reply:
Use the more general typing I suggested above. I don't think the Union adds anything.

Reply:
It is done, I think we are ready to go.